Sources/Markdown/Markdown.docc/Markdown/BlockMarkup.md

```diff
@@ -0,0 +1,34 @@
+# Block Nodes of Markup
+
+Block Nodes of Markup
+
+## Overview
+
+Block Nodes of Markup
```

Comment:
For a lot of these files, if the abstract and overview are just the same as the title, you can get away with leaving them out entirely, for example:

```markdown
# Markup Block Nodes

## Topics

### Block Container Blocks

...
```

Would you be willing to update these articles to remove these duplicate paragraphs?